src/constants.ts

```diff
@@ -10,3 +10,20 @@ export const CONCEPT_STATES = {
 };
 
 export const PAGE_SIZE = 20;
+
+export const TEMPLATE_RELATION_TYPES = {
```

Comment:
Were these types hard coded before? What if we want to add a new type?

Reply:
These are hardcoded in backend also(models, forms) for rendering in templates. If we need to add new type, it requires code change, handling cases, etc. anyways from backend.